framework/branch-key-store.md

```diff
@@ -601,8 +601,8 @@ List of TransactWriteItem:
     - “enc” (B): the wrapped DECRYPT_ONLY Branch Key `CiphertextBlob` from the KMS operation
     - “create-time” (S): `timestamp`
     - "kms-arn" (S): configured KMS Key
-    - “hierarchy-version” (N): 1
-    - Every key-value pair of the custom [encryption context](./structures.md#encryption-context-3) that is associated with the branch key
+    - “hierarchy-version” (N): either `1` or `2`, depending on the `hierarchy-version`
+    - Every key-value pair in the [encryption context](./structures.md#encryption-context) provided by users and is associated with the branch key
```

Comment:
This is version key, so it is slightly different.

What is the "encryption context provided by users" at this point? I think we want to be careful at this point. Can we just refer to the branch key context here? And say "Just write the whole context like this"? Because that will already have anything prepended correct?